provision/provision.sh

```diff
@@ -764,7 +764,7 @@ services_restart() {
   phpdismod xdebug
 
   # Enable PHP mailcatcher sendmail settings by default
```

Comment:
can you replace also mailcatcher here with mailhog?

Reply:
done